### PR TITLE
Consume content metrics

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java [hunk 1]
```diff
@@ -56,104 +57,120 @@ private MetricsResponse(Slime response, NodeList applicationNodes, NodeRepositor
...
+                               "content.proton.documentdb.matching.queries.rate");
+            }
         },
         writeRate { // writes per second
-            public String metricResponseName() { return "feed.http-requests.rate"; }
-            double convertValue(double metricValue) { return (float) metricValue; }
-            double defaultValue() { return 0.0; }
+            public List<String> metricResponseNames() {
+                return List.of("feed.http-requests.rate",
+                               "vds.filestor.alldisks.allthreads.put.sum.count.rate",
```
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java:154]: Should these be added to `AutoscalingMetrics.java`?

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java [hunk 2]
```diff
@@ -56,104 +57,120 @@ private MetricsResponse(Slime response, NodeList applicationNodes, NodeRepositor
...
+            }
         },
         inService {
-            public String metricResponseName() { return "in_service"; }
-            double convertValue(double metricValue) { return (float)metricValue; } // Really a boolean
-            double defaultValue() { return 1.0; }
+            public List<String> metricResponseNames() { return List.of("in_service"); }
+            double computeFinal(List<Double> values) {
+                // Really a boolean. Default true. If any is oos -> oos.
+                return values.stream().anyMatch(v -> v == 0) ? 0 : 1;
```
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java:142]: Should we return 0 if `values` is empty?

**Review by freva — APPROVED** [review]

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java (hunk 1 above)
Comment (bratseth) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java:154]: Yes, i forgot - thanks!

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java (hunk 2 above)
Comment (bratseth) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java:142]: Not all services send this (yet) so it needs to default to 1.

**Pull request merged**